### ISSUE TITLE
some problems about  dictionary

### ISSUE THREAD

**Issue by zhaoxiaofenfen** [lurenjia212/Copmare_ECG_CS#2]
hello author,thank you for your codes.I have some problems ,someone is thatYou provided a dictionary, but no dictionary training file. Can you give me a source code for dictionary training? I trained one myself, but the results are not as good as yours. Another problem is that you mentioned that the dictionary is used for dictionary training. The training test set only contains QRS waves, I wonder if this means that QRS waves are extracted without other data? I am a little confused about the division of your test set and training set, I hope you can enlighten me.Thank you
